CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Docs: Add versioning of GitHub Pages and pyauditor docs (#551) ([@QuantumDancer](https://github.com/QuantumDancer)
 - Docs: Add overview of API endpoints (#597) ([@QuantumDancer](https://github.com/QuantumDancer)
 - Apel plugin: Add optional config setting for style of summary message ([@dirksammel](https://github.com/dirksammel))
+- Docs: Add removal of get_[started/stopped]_since api endpoint (#623) ([@raghuvar-vijay](https://github.com/raghuvar-vijay))
```

Comment:
Instead of this line, can you maybe add an item to the "Breaking changes" section in the changelog that just contains something like this: "API: Deprecate `get_[started/stopped]_since` endpoint". And maybe also mention the original PR (#525) instead of #623 

(I'm just noticing that we should put all breaking changes in the corresponding section in the changelog. I will do this for the other changes once I prepare the release.)